contracts/test/CoveragePoolStub.sol

```diff
@@ -1,10 +1,35 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.4;
 
+import "../CoveragePoolConstants.sol";
+
 contract CoveragePoolStub {
+    uint256 public covTotalSupply;
+
     event FundsSeized(address indexed recipient, uint256 portionToSeize);
+    event AssetPoolSharesGranted(address indexed recipient, uint256 covAmount);
 
     function seizeFunds(address recipient, uint256 portionToSeize) external {
         emit FundsSeized(recipient, portionToSeize);
     }
+
+    function grantAssetPoolShares(address recipient, uint256 covAmount)
+        external
+    {
+        emit AssetPoolSharesGranted(recipient, covAmount);
+    }
+
+    function covAmountToGrant(uint256 portionToGrant)
```

Comment:
Do we have to implement this function here?

Reply:
Yep, some tests are using the coverage pool stub and look for that method.

Reply:
Hmm.... I commented out this function and run all tests (unit + system) and all is green. If tests use this function then something is probably wrong because it returns 0 now (`covTotalSupply` is zero).

Reply:
Ah I see now, it was needed before but it's no longer the case. See https://github.com/keep-network/coverage-pools/pull/111/commits/e7a6f40c79bfcfd0bd82e56ef102f6a282e78380